odtp/cli/db.py

```diff
@@ -9,10 +9,22 @@
 
 app = typer.Typer()
 
+@app.command()
+def ls(collection_name: str):
```

Comment:
Actually I was thinking to discuss the implementation of the `Argument` datatype instead of `Option`.  

In this [thread](https://softwareengineering.stackexchange.com/questions/366218/positional-arguments-vs-options-in-a-command-line-interface) I like the answer of leaving the positional argument for those parameters that are required: 

> Most places I’ve seen use both. Position based parameters for required parameters, and options based parameters for optional parameters.

In this case the collection is mandatory so it's always used. I would like to propose extending this to the rest of the CLI if we have time.